src/rules/property-shorthand-no-unnecessary-value/README.md

```diff
@@ -0,0 +1,58 @@
+# property-shorthand-no-unnecessary-value
+
+Disallow longhand form for properties that support it.
+
+```css
+.foo { margin: 1px 1px 1px 1px; }
+/**            ↑
+ *  These properties */
+```
+
+The rule warns when you use longhand form for properties:
```

Comment:
How about `This rule warns you when you use redundant values in the following longhand properties:`